.github/actions/upload-s3/action.yml

```diff
@@ -0,0 +1,35 @@
+name: Upload S3
+
+description: Upload S3
+
+inputs:
+  aws-region:
+    required: true
+    type: string
+  role-to-assume:
+    required: true
+    type: string
+  s3-bucket:
+    required: true
+    type: string
+  path:
+    required: true
+    type: string
+
+runs:
+  using: composite
+  steps:
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v4
```

Comment:
does this need to be a hash version?

Reply:
But this one isn't an official _Github_ action. I thought the MetaMask security team had restrictions in place to never trust versioned actions, unless they were from Github. Maybe that's not the case anymore (or maybe it doesn't apply to this repo, which is probably an accident)

Reply:
Oh, and I don't think `@v4` references a branch; I think it just references a git tag.